solr/core/src/java/org/apache/solr/handler/SchemaHandler.java

```diff
@@ -206,6 +206,10 @@ private void handleGET(SolrQueryRequest req, SolrQueryResponse rsp) {
     }
   }
 
+  /**If a plugin is loaded from a package, the version of the package being used should be added
```

Comment:
minor: nor formatted well (and same goes for many other added javadocs)
I love the new comment though